MovieNight/ActorViewController.swift

```diff
@@ -11,22 +11,53 @@ import UIKit
 class ActorViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
 
     @IBOutlet weak var actorTableView: UITableView!
+    var apiClient = MovieNightApiClient()
+    var actors = [Actor]()
+    var hasNextPage: Bool = true {
+        didSet {
+            self.page += 1
+        }
+    }
+    var page = 1
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         actorTableView.delegate = self
         actorTableView.dataSource = self
+        
+        loadData()
+    }
+    
+    func loadData() {
+        fetchForActors(with: page)
+    }
+    
+    func fetchForActors(with page: Int) {
+        apiClient.fetchForActors(page: page) { [weak self] (result) in
+            switch result {
+            case .failure(let error) :
+                return print(error)
+            case .success(let resource, let hasPage):
+                self?.actors += resource
+                self?.hasNextPage = hasPage
+                self?.actorTableView.reloadData()
+            }
+        }
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "actorIdCell", for: indexPath) as! ActorTableViewCell
         
+        if (actors.count - indexPath.row) == 5 && hasNextPage {
```

Comment:
You could put this condition in a method to make it more readable. Something like

`if (shouldFetchNextPage())`